doc/pow/pow.md

```diff
@@ -1,43 +1,34 @@
-# Grin's Proof-of-Work
```

Comment:
I think the changes in this doc are not relevant to the difficulty adjustment algorithm. Maybe we should have them in a separate PR instead?

Reply:
the doc fixes don't need review so this seemed the easiest way to get then applied.